projects/10up-theme/includes/blocks/example-block/markup.php

```diff
@@ -27,6 +27,8 @@
 ?>
 <div class="<?php echo esc_attr( $args['class_name'] ); ?>">
 	<h2 class="wp-block-example-block__title">
-		<?php echo wp_kses_post( $args['attributes']['customTitle'] ); ?>
+		<?php if ( isset( $args['attributes']['customTitle'] ) ): ?>
```

Comment:
@nicholasio why are you making this change here? 🤔 are there any instances where you encountered this being undefined? 

Reading the code it should not be possible to be undefined

Reply:
when I added the block and saved without adding a title there wasn't a default title

Reply:
@fabiankaegy maybe there's a bug elsewhere? anyway this is just a test project, I'm not too concerned about this :)